confidant/routes/v1.py

```diff
@@ -778,6 +819,12 @@ def update_credential(id):
     cipher = CipherManager(data_key['plaintext'], version=2)
     credential_pairs = cipher.encrypt(update['credential_pairs'])
     update['metadata'] = data.get('metadata', _cred.metadata)
+    update['group'] = data.get('group')
+    # if using groups for access control, check that the user is a member
+    # of the new group. Setting the group to empty is always permitted
+    if app.config.get('USE_GROUPS'):
+        if update['group'] and not authnz.user_is_member(update['group']):
+            return jsonify({'error': 'Must be a member of the destination group'}), 400
```

Comment:
This logic seems backwards to me. If you're able to add a check for the `_cred.group` value that would be great (same as `update['group']` there I believe), otherwise I'm not sure this is useful to have